content/managing-apps.md

```diff
@@ -0,0 +1,346 @@
+  - Remove
+  - Delete
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: pop
+tableOfContents: true
+---
+
+## Installing & Uninstalling Applications via Terminal
+
+Launch the Terminal `SUPER` + `T` to Install applications using a command line interface. Terminal commands instruct package managers to retrieve application installation packages from repositories.
+
+### Flatpaks & .Deb Files
+
+**Flatpaks** will install a “containerized” version of the software. This means the software runs in its own sandbox, and the installation will include all dependencies and libraries required by the application. Flatpaks pull all dependencies from flathub.org, while installing with apt will pull from Ubuntu repos.
+
+Substitute `[packagename]` with the desired application (do not include brackets in the command).
+
+Apt install:
+
+```bash
+apt install [packagename]
+```
+
+Apt uninstall:
+
+```bash
+sudo apt remove [packagename]
+```
+
+Apt uninstall (alternative):
+
+```bash
+sudo apt purge [packagename]
+```
+
+Flatpak install:
+
+```bash
+flatpak install [packagename]
+```
+
+Flatpak uninstallL
+
+```bash
+flatpak uninstall [packagename]
+```
+
+## Updating POP!\_OS via Terminal
+
+1. Open a terminal by pressing `SUPER` + `T` and type the following commands:
+
+```bash
+sudo apt update
+```
+
+```bash
+sudo apt dist-upgrade
+```
+
+2. Answer `Y` and enter the password when prompted.
+
+![yes](/images/managing-apps/IURA_22_24.png)
+
+3. If an error appears, read the error message and run any commands it suggests.
+
+![error](/images/managing-apps/IURA_22_25.png)
+
+## Pop!\_OS 24.04
+
+### Installing an Application
+
+Cosmic includes a graphical user interface (GUI) called the Cosmic Store for easy installation and management of open source applications.
+
+1. Open Cosmic Store by clicking the icon in the dock, or press `SUPER` and type “Cosmic Store”.
+
+![cosmic store icon](/images/managing-apps/IURA_24_1.png)
+
+2. Explore apps or type the application name in the search field.
+
+![cosmic store search](/images/managing-apps/IURA_24_2.png)
+
+3. Select the system or flathub version.
+
+![app download type version](/images/managing-apps/IURA_24_3.png)
+
+### Installing an Application Not Included in the Cosmic Store
+
+The Cosmic Store does not include all applications available for installation in Pop!_OS. Use the web to search for and install additional applications.
+
+1. Search for the application download link and instructions online.
+
+![cosmic app google search](/images/managing-apps/IURA_24_4.png)
+
+2. Download the application. When given file options, choose 64 bit .deb as the file type.
+
+![cosmic deb file on download](/images/managing-apps/IURA_22_5.png)
+
+3. After downloading, go to the Downloads folder and double click the .deb file.
+
+![cosmic store icon](/images/managing-apps/IURA_24_5.png)
+
+4. Cosmic Store will open automatically and present you with the option to install the file.
```

Comment:
The word "automatically" is superfluous and arguably wrong (you double-clicked on the file manually to make it open). It's probably best to remove that word.